en/ros/README.md

```diff
@@ -9,6 +9,26 @@ This section contains topics about using ROS for offboard control with PX4.
 ROS is only officially supported on Linux platforms.
 :::
 
+# Robotics using ROS 2
+
+[ROS 2](https://index.ros.org/doc/ros2/) is the newest version of ROS.
+It captures most of the learnings and recently added features of ROS (1), improving a number of flaws of the earlier version.
+
+The translation layer between ROS 2 and PX4 is software known as the [PX4-Fast RTPS(DDS) Bridge](../middleware/micrortps.md).
```

Comment:
@TSC21 Did minor update to this, changing "MicroRTPS bridge" to "PX4-Fast RTPS(DDS) Bridge"

**Question**
- the installation section that follows appears to be for ROS (1) but the implication is that is is for ROS2 from the heading leevesl - and my thinking was that PX4-ROS 2 bridge is where you talk about ROS2 setup.

Reply:
> @TSC21 Did minor update to this, changing "MicroRTPS bridge" to "PX4-Fast RTPS(DDS) Bridge"

Here it's fine, and also as the title of the micrortps.md. The microRTPS name should be user othwerwise.

> the installation section that follows appears to be for ROS (1) but the implication is that is is for ROS2 from the heading leevesl - and my thinking was that PX4-ROS 2 bridge is where you talk about ROS2 setup.

It's for both. It's given the option of also user ROS(1) through the ros1_bridge.